.markdownlint.yaml

```diff
@@ -0,0 +1,275 @@
+# Refer to https://github.com/DavidAnson/markdownlint for additional rules
+
+# Set default state for rules to false in order to handle markdownlint new rule additions
+default: false
+
+# MD001/header-increment - Heading levels should only increment by one level at a time
+MD001: true
+
+# MD003/header-style - Heading style
```

Comment:
Heh, comments like these (which are absolutely necessary btw) are why I'm so happy I opted for human readable linter rule names in Regal.